tuning/autotune.py

```diff
@@ -185,6 +196,11 @@ def get_benchmark_time(self) -> Optional[float]:
         except ValueError:
             return None
 
+    def generate_sample_result(
+        self, candidate_id: int = 0, mean_time: float = random.uniform(100.0, 500.0)
+    ) -> str:
+        return f"{candidate_id}\tMean Time: {mean_time:.1f}\n" # time in ms
```

Comment:
Isn't the time unit implicit and dependent on the output of iree-benchmark-module? It won't always be ms.